server/game/player.js

```diff
@@ -310,7 +310,14 @@ class Player extends Spectator {
             card.moveTo('draw deck');
             this.drawDeck.push(card);
         });
+        this.cardsInPlay.each(card => {
```

Comment:
While we're at it, I'd appreciate if we can rename/generalize this method into something like resetDrawDeck(), if not even resetDecks() (plural) makingit also take care of the plot deck. I plan to use this to implement a /reset chat command, which we've discussed before, and it'd be nice to have most of its logic part of a method also used for other purposes, such as game initialization at startup.